apps/etterlatte-klage/src/main/kotlin/no/nav/etterlatte/klage/modell/KabalHjemmel.kt

```diff
@@ -0,0 +1,182 @@
+package no.nav.etterlatte.klage.modell
+
+// Direkte kopiert fra Kabals kodeverk: https://github.com/navikt/klage-kodeverk/blob/main/src/main/kotlin/no/nav/klage/kodeverk/hjemmel/Hjemmel.kt
+// Trur vi berre treng eit subsett av dette
+
+enum class KabalHjemmel(
+    val id: String,
+    val lovKilde: LovKilde,
+    val spesifikasjon: String
+) {
+    // @formatter:off
+
+    // TODO give more explaining enum-names or improve Swagger-docs.
+
+    BTRL_2("619", LovKilde.BARNETRYGDLOVEN, "§ 2"),
```

Comment:
Hvorfor importerer vi ikke modulen deres i stedet for å duplisere og risikere å komme ut av synk her?

Reply:
Hypotesa (mi) er at vi berre treng eit subsett av desse, og at vi kan ta bort alt anna. Så da vil denne gi meir meining.